src/SDK/Language/CLI.php

```diff
@@ -74,6 +74,12 @@ public function getFiles()
                 'template'      => 'cli/package.json.twig',
                 'minify'        => false,
             ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'appwrite.json',
```

Comment:
Can we name it something else? We use the file `appwrite.json` for project configuration used by Appwrite commands (`init, deploy`). It cold create some confusions when looking at CLI SDK

Reply:
My idea: `scoop.json`

Reply:
I thought the same thing but since the manifest doesn't contain a name the name of the file is the name that it installs the program under. I tried using scoop.json but when I tried it actually uninstalled scoop because it thought this was a new version of it. So to have the program called appwrite it has to be appwrite.json. A little annoying but I think if we keep it in a folder called scoop it could work

Reply:
Scoop folder would be good.